runtime/src/lib.rs

```diff
@@ -426,6 +446,174 @@ impl pallet_local_treasury::Config for Runtime {
     type Event = Event;
 }
 
+impl pallet_saft_registry::Config for Runtime {
+    // Using root as the admin origin for now
+    type AdminOrigin = frame_system::EnsureRoot<AccountId>;
+    type Event = Event;
+    type Balance = Balance;
+    type AssetRecorder = AssetIndex;
+    type AssetId = AssetId;
+}
+
+parameter_types! {
+    pub const ProposalSubmissionPeriod: <Runtime as frame_system::Config>::BlockNumber = 10;
+    pub const VotingPeriod: <Runtime as frame_system::Config>::BlockNumber = 5;
+}
+
+ord_parameter_types! {
+     pub const MinCouncilVotes: usize = 4;
+}
+
+impl pallet_committee::Config for Runtime {
+    type ProposalSubmissionPeriod = ProposalSubmissionPeriod;
+    type VotingPeriod = VotingPeriod;
+    type MinCouncilVotes = MinCouncilVotes;
+    // Using root as the admin origin for now
+    type ProposalSubmissionOrigin = frame_system::EnsureRoot<AccountId>;
+    type ProposalExecutionOrigin = frame_system::EnsureRoot<AccountId>;
```

Comment:
We should open an issue for checking/organizing the config of `Origin` for pallets

Reply:
follow up on #64